### PR TITLE
AWS Quick Start concepts

### PR DESCRIPTION
**Pull request by seglo**
A follow up for the quick start guide #706 based on feedback from @mckeeh3

### CODE REVIEW

**Review by RayRoestenburg — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc [hunk 1]
```diff
@@ -81,31 +97,95 @@ Choose an AWS region to use:
...
+Preview and provision the playbook.
 
 [source,shell script]
 ----
 pulumi up
 ----
 
-If the provisioning is successful then all the pertinent details of the cluster will be output to the console (except for passwords).
-You can copy & paste this configuration, or access it at the command line with `pulumi stack output <OUTPUT KEY>`.
+While the cluster is provisioning you will be informed in the terminal about which resources are in the process of being created or have been created.
```
Comment (thothmoses) [inline docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc:115]:
```suggestion
While the cluster is provisioning you will be informed, in the terminal, about which resources are in the process of being created or have been created.
```

File: docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc [hunk 2]
```diff
@@ -81,31 +97,95 @@ Choose an AWS region to use:
...
+If using the the Pulumi service for state then the `pulumi up` command will include a link to "View Live" the progress of the deployment.
```
Comment (thothmoses) [inline docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc:116]:
```suggestion
If you are using the Pulumi service for the state then the `pulumi up` command will include a link to "View Live" the progress of the deployment.
```

File: docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc [hunk 3]
```diff
@@ -81,31 +97,95 @@ Choose an AWS region to use:
...
+The `pulumi up` command will exit when all resources are fully deployed and online.
 
-[NOTE]
-====
-Kafka and JDBC Kubernetes secrets are created in the Akka Cloud Platform Operator namespace.
-These must be copied to each microservice namespace that references them.
-Here is one way to copy a resource from one namespace to another:
+If a failure occurs then the command will exit early and inform you of the reason.
+When this occurs and you can correct the issue yourself then rerun `pulumi up` to resume updating where you left off.
+If the error can't be resolved then you can use the `pulumi destroy` command to rollback any partially deployed resources.
```
Comment (thothmoses) [inline docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc:121]:
```suggestion
If the error can't be resolved, you can use the `pulumi destroy` command to roll back any partially deployed resources.
```

File: docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc [hunk 4]
```diff
@@ -81,31 +97,95 @@ Choose an AWS region to use:
...
+
+If the deployment is successful then all the pertinent details of the cluster will be output to the console (except for passwords).
```
Comment (thothmoses) [inline docs-source/docs/modules/deployment/pages/aws-install-quickstart.adoc:123]:
```suggestion
If the deployment is successful, all of the pertinent details of the cluster will be output to the console (except for passwords).
```

**Review by thothmoses — CHANGES_REQUESTED** [review]
A few tweaks

**Comment by seglo** [comment]
Don't merge this yet. Making a few more changes.

**Review by thothmoses — APPROVED** [review]
Excellant.

**Pull request merged**